tests/TemporaryBranchName.cs

```diff
@@ -0,0 +1,23 @@
+namespace renovate_config.tests;
+
+internal abstract class TemporaryBranchName
+{
+    public const string DateFormat = "yyyyMMddHHmmss";
+    public static readonly TimeSpan MaximumAge = TimeSpan.FromHours(2);
```

Comment:
I think we can allow more time, so if a CI fails during the night we can still investigate in the morning.

```suggestion
    public static readonly TimeSpan MaximumAge = TimeSpan.FromHours(24);
```

Reply:
Works for me! Previously I tried so many values... 3 days, 1 day, 12 hours, 6 hours... it's about finding the right balance for troubleshooting and not too many branches!